docs/guides/oracle/restart/restart.md

```
@@ -0,0 +1,297 @@
+title: Restart Oracle
+menu:
+  docs_{{ .version }}:
+    identifier: guides-oracle-restart-details
+    name: Restart Oracle
+    parent: guides-oracle-restart
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)